gitops-wg/interested-parties.md

```diff
@@ -54,8 +60,6 @@ Intuit | Edward Lee
 Intuit | Henrik Blixt
 Intuit | Alexander Matyushentsev
```

Comment:
@wojtekidd Also need to update Alex's entry. cc @alexmt

Reply:
Yes please

Reply:
Done